aci/resource_aci_rest_managed.go

```diff
@@ -119,9 +119,16 @@ func getAciRestManaged(d *schema.ResourceData, c *container.Container, expectObj
 		return diag.Errorf("Failed to retrieve REST payload for class: %s.", className)
 	}
 
+	var setContentAnnotation bool
+	if _, ok := contentStrMap["annotation"]; ok {
+		setContentAnnotation = true
+	}
+
 	for attr, value := range restContent {
 		// Ignore certain attributes
-		if !containsString(IgnoreAttr, attr) {
+		if attr == "annotation" && setContentAnnotation {
+			newContent[attr] = value.(string)
+		} else if attr != "annotation" && !containsString(IgnoreAttr, attr) {
```

Comment:
any particular reason this is else if and not else ?

Reply:
Yes because it needs to be added only when a certain condition is met. In this case that would be the following:

1. `attr != "annotation"` - because in the if condition setContentAnnotation (used to determine if annotation is set in the content input map) is set to False the else statement could have scenario where `attr == "annotation"` thus we need to check again
2. !containsString(IgnoreAttr, attr) - because certain attributes should not be set in content, if else it would add all attributes and the conditional would not be needed